### PR TITLE
Modify Corporate Standard Char Of Account and 2017 Small And Medium-s…

### PR DESCRIPTION
**Pull request by Colinliz**
…ized Enterprises Accounting Chart By PSCloud

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-<odoo noupdate="1">
-    <record id="l10n_chart_china_small_business" model="account.chart.template">
-        <field name="property_account_receivable_id" ref="l10n_cn_1122"/>
-        <field name="property_account_payable_id" ref="l10n_cn_2202"/>
-        <field name="property_account_expense_categ_id" ref="l10n_cn_6401"/>
-        <field name="property_account_income_categ_id" ref="l10n_cn_6001"/>
-        <field name="income_currency_exchange_account_id" ref="l10n_cn_6051"/>
-        <field name="expense_currency_exchange_account_id" ref="l10n_cn_6711"/>
-    </record>
-</odoo>
```
Comment (GoodERPJeff) [inline addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml]: 这个不需要么？

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml [hunk 2]
```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo noupdate="1">
-    <record id="l10n_chart_china_small_business" model="account.chart.template">
-        <field name="property_account_receivable_id" ref="l10n_cn_1122"/>
-        <field name="property_account_payable_id" ref="l10n_cn_2202"/>
-        <field name="property_account_expense_categ_id" ref="l10n_cn_6401"/>
```
Comment (GoodERPJeff) [inline addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml]: 应该是 1401

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml (hunk 1 above)
Comment (GoodERPJeff) [reply addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml]: 跟jeffery确认了，这个需要的，改一下

**Comment by pedrobaeza** [comment]
Please communicate here in English, as this has global audience.

File: addons/l10n_cn_small_business/data/account.account.template.csv [hunk 3]
```diff
@@ -1,76 +1,206 @@
...
+"small_business2017_chart5603002","5603002","利息收入","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5711","5711","营业外支出","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5711001","5711001","坏账损失","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5711002","5711002","无法收回的长期债券投资损失","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5711003","5711003","无法收回的长期股权投资损失","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5711004","5711004","自然灾害等不可抗力因素造成的损失","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5711005","5711005","税收滞纳金","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart5801","5801","所得税费用","False","data_account_type_general","l10n_cn_small_business.l10n_chart_china_small_business"
```
[the remark this answers is not in the document]
Comment (Colinliz) [reply addons/l10n_cn_small_business/data/account.account.template.csv]: Ok,i have added a blank line at the bottom

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml (hunk 1 above)
Comment (Colinliz) [reply addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml]: II'm done with it.

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml (hunk 2 above)
Comment (Colinliz) [reply addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_post_data.xml]: I've changed "l10n_cn_6401" to "l10n_cn_1401"

**Comment by jeffery9** [comment]
@mat-e  please help to merge this pr.

File: addons/l10n_cn_small_business/data/account_tax_template_data.xml [hunk 4]
```diff
@@ -1,154 +1,65 @@
...
-        <field name="tax_group_id" ref="l10n_cn.l10n_cn_tax_group_vat_11"/>
-    </record>
-    <record id="l10n_cn_small_business_purchase_excluded_6" model="account.tax.template">
+
+    <record id="vats_small_business_13" model="account.tax.template">
         <field name="chart_template_id" ref="l10n_chart_china_small_business"/>
-        <field name="name">税收6％ - 中国小企业会计科目表</field>
-        <field name="description">税收6％</field>
-        <field name="amount">6</field>
+        <field name="name">13％销项税 - 中国小企业会计科目表</field>
```
Comment (jeffery9) [inline addons/l10n_cn_small_business/data/account_tax_template_data.xml]: switch name and description,  name should shorter than description.
the same for all  tax.

File: addons/l10n_cn_small_business/data/account_tax_template_data.xml [hunk 5]
```diff
@@ -1,154 +1,65 @@
...
-        <field name="price_include" eval="0"/>
-        <field name="tax_group_id" ref="l10n_cn.l10n_cn_tax_group_vat_17"/>
-    </record>
-    <record id="l10n_cn_small_business_sales_excluded_11" model="account.tax.template">
+    <record id="vats_small_business_16" model="account.tax.template">
         <field name="chart_template_id" ref="l10n_chart_china_small_business"/>
-        <field name="name">税收11％ - 中国小企业会计科目表</field>
-        <field name="description">税收11％</field>
-        <field name="amount">11</field>
+        <field name="name">16％销项税 - 中国小企业会计科目表</field>
```
Comment (jeffery9) [inline addons/l10n_cn_small_business/data/account_tax_template_data.xml]: name should shorter than description, better  display in list view.

File: addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml [hunk 6]
```diff
@@ -2,14 +2,23 @@
...
     <data noupdate="1">
+        <record id="account_1003" model="account.account.template">
+            <field name="name">转让帐户</field>
+            <field name="code">1003</field>
+            <field name="reconcile" eval="True" />
+            <field name="user_type_id" ref="account.data_account_type_current_assets" />
+        </record>
         <!-- Chart template -->
         <record id="l10n_chart_china_standard_business" model="account.chart.template">
             <field name="name">中国会计科目表 （财会[2006]3号《企业会计准则》</field>
```
Comment (joshuajan) [inline addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml]: [TYPO] There should be a '）' at the end.  :-)

File: addons/l10n_cn_standard/__manifest__.py [hunk 7]
```diff
@@ -6,14 +6,17 @@
...
 {
     'name': 'China - Standard CoA',
-    'version': '2.0',
+    'version': '12.0.1.0',
     'category': 'Localization',
-    'author': ['lvxin@gmail.co', 'oldrev@gmail.co', 'ccdos@intoerp.com', 'jeff@osbzr.com', 'jeffery9@gmail.com'],
+    'author': ['lvxin@gmail.co', 'oldrev@gmail.co', 'ccdos@intoerp.com', 'jeff@osbzr.com', 'jeffery9@gmail.com',
+               'pscloud@inspur.com'],
 
-    'website': 'http://shine-it.net',
```
Comment (digitalsatori) [inline addons/l10n_cn_standard/__manifest__.py]: please keep Shine IT (先安科技）as the contributor

File: addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml (hunk 6 above)
Comment (Colinliz) [reply addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml]: You are very careful～

Comment (leangjia) [reply addons/l10n_cn_standard/data/l10n_cn_standard_chart_data.xml]: Good job.@joshuajan

**Comment by leangjia** [comment]
I needs it.

**Comment by GoodERPJeff** [comment]
@mat-e it's ready for merge

**Comment by mart-e** [comment]
Hi,

Sorry but it is not. The branch has conflicts and is making changes that I don't think are suitable for a stable branch (removing records,...).
Also, please explain what changes you are doing to specify in the commit message. See [our guidelines](https://www.odoo.com/documentation/12.0/reference/guidelines.html#commit-message-structure) about it (don't explain just what, also explain why it is important to change that).

Also @jco-odoo is responsible for the localisations so should review the changes before merging.

**Comment by Colinliz** [comment]
@jco-odoo 
Hi,
Can you help to review the changes before merging?
I have  modified small and medium-sized enterprises accounting chart, and deleted included and excluded tax: 17% 11% 6% 3%, added included and excluded tax: 13% 16%.

**Comment by jco-odoo** [comment]
@Colinliz Nice to meet you on Github.  The changes are indeed quite extensive.  You just changed the Chart of Accounts with yours.  Did you check if there are compatibilities with the old one?  (if the differences are really that huge?)

For the taxes: when did the taxes of 17, 11, 6, 3% change?  Because if it is not too long ago, it might be interesting to keep them for the moment.  (check also what should be the tax by default, should be 16% I suppose)  If I check online, the 6% vat is still very much active.  

As there is no connection with any tags, there might be a way to do this in a stable version, but then we have to be sure it is a real improvement.  Maybe it is better to do something solid with tax reports, ... in master.

**Comment by GoodERPJeff** [comment]
> @Colinliz Nice to meet you on Github. The changes are indeed quite extensive. You just changed the Chart of Accounts with yours. Did you check if there are compatibilities with the old one? (if the differences are really that huge?)
> 
> For the taxes: when did the taxes of 17, 11, 6, 3% change? Because if it is not too long ago, it might be interesting to keep them for the moment. (check also what should be the tax by default, should be 16% I suppose) If I check online, the 6% vat is still very much active.
> 
> As there is no connection with any tags, there might be a way to do this in a stable version, but then we have to be sure it is a real improvement. Maybe it is better to do something solid with tax reports, ... in master.

China VAT just change to ：13%，9%，6%，0%。
please change the xml file  @Colinliz 
other change is ok to me

**Comment by Colinliz** [comment]
@jco-odoo Thank you very much for your suggestion.I have checked that my revised account chart is fully compatible with the old one, and it also applies to small businesses today.

For the taxes:I keep 13% 9% and 6% account tax, and remove 
17% 11% and 3%. After discussion I think 13% account tax should be default, because China has already changed 16% tax to 13% tax.

It's hard for you to review my code again and give your opinion. I'll revise it in time.

**Comment by Colinliz** [comment]
@jco-odoo Sorry to bother you, but do you have time to review the code for me?

File: addons/l10n_cn_small_business/data/account.account.template.csv [hunk 8]
```diff
@@ -1,76 +1,206 @@
...
-"l10n_cn_6701","Assets impairment Loss","6701","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6711","Non-operating expenses","6711","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6801","Income Tax Expense","6801","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
-"l10n_cn_6901","Prior year income adjustment","6901","account.data_account_type_expenses","l10n_cn_small_business.l10n_chart_china_small_business","False"
+"id","code","name","reconcile","user_type_id/id","chart_template_id/id"
+"small_business2017_chart1001","1001","库存现金","False","account.data_account_type_liquidity","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart1002","1002","银行存款","False","account.data_account_type_liquidity","l10n_cn_small_business.l10n_chart_china_small_business"
+"small_business2017_chart1012","1012","其他货币资金","False","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business"
```
Comment (jco-odoo) [inline addons/l10n_cn_small_business/data/account.account.template.csv]: Why change xml id?  It is breaking stuff for breaking stuff.  If you kept l10n_cn_ and the same structure, we could really see the difference between the two.

Comment (jco-odoo) [reply addons/l10n_cn_small_business/data/account.account.template.csv]: And sometimes you have a code of length 7, while you put code digits at 4

File: addons/l10n_cn_small_business/data/account_tax_template_data.xml [hunk 9]
```diff
@@ -1,154 +1,96 @@
...
-        <field name="amount">17</field>
-        <field name="amount_type">percent</field>
-        <field name="refund_account_id" ref="l10n_cn_2221"/>
-        <field name="account_id" ref="l10n_cn_2221"/>
-        <field name="type_tax_use">sale</field>
-        <field name="price_include" eval="1"/>
-        <field name="tax_group_id" ref="l10n_cn.l10n_cn_tax_group_vat_17"/>
+    <!-- Account Tax Tags-->
+    <record id="tax_tag1" model="account.account.tag">
+        <field name="name">6％销项税</field>
```
Comment (jco-odoo) [inline addons/l10n_cn_small_business/data/account_tax_template_data.xml]: What do you need the tags for?

**Comment by jco-odoo** [comment]
@Colinliz I would say as the VAT rates (16% -> 13%) came into force on the first of April, it can be a good idea to keep them if people still need to create an invoice for March e.g., but we don't have the 16% in the chart, only the 17%.  

Is not 13% automatically already the default tax?  I agree with @mart-e that some context can help us.  We can always do a call if you want.

File: addons/l10n_cn_small_business/data/account.account.template.csv (hunk 8 above)
Comment (GoodERPJeff) [reply addons/l10n_cn_small_business/data/account.account.template.csv]: @jco-odoo  code digit define the "minimal" length of account code, it keep adding trialling 00 to first level account code when it is "6". change it to "4" is the must we have from 10 years ago in openerp.

Yes, xml id should be changed back @Colinliz

**Comment by GoodERPJeff** [comment]
> @Colinliz I would say as the VAT rates (16% -> 13%) came into force on the first of April, it can be a good idea to keep them if people still need to create an invoice for March e.g., but we don't have the 16% in the chart, only the 17%.
> 
> Is not 13% automatically already the default tax? I agree with @mart-e that some context can help us. We can always do a call if you want.

16% will be only been hardly used in Apil 2019, no reason keep it in our chart.

File: addons/l10n_cn_small_business/data/account.account.template.csv [hunk 10]
```diff
@@ -1,76 +1,206 @@
-"id","name","code","user_type_id/id","chart_template_id/id","reconcile"
-"l10n_cn_1012","Other Monetary Funds","1012","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
```
Comment (jco-odoo) [inline addons/l10n_cn_small_business/data/account.account.template.csv]: It is of course logical that the CoA is in the language of the country itself, but it is quite a heavy change for stable.  On the other hand, the l10n_cn module is multilang, so you could add the translation to English.  (I suppose there is a reason it was originally in English)

File: addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml [hunk 11]
```diff
@@ -13,16 +13,16 @@ http://kjs.mof.gov.cn/zhengwuxinxi/zhengcefabu/201111/t20111107_605525.html
...
+
         <record id="l10n_chart_china_small_business" model="account.chart.template">
             <field name="name">小企业会计科目表（财会[2011]17号《小企业会计准则》）</field>
-            <field name="code_digits" eval="6"/>
+            <field name="code_digits" eval="4"/>
             <field name="currency_id" ref="base.CNY"/>
             <field name="cash_account_code_prefix">1001</field>
             <field name="bank_account_code_prefix">1002</field>
             <field name="transfer_account_code_prefix">1012</field>
             <field name="spoken_languages" eval="'en_US'"/>
```
Comment (jco-odoo) [inline addons/l10n_cn_small_business/data/l10n_cn_small_business_chart_data.xml]: By changing the CoA, you change to Chinese here, but as said before, you can add both.

**Comment by jco-odoo** [comment]
If you do export translations with "new language" and "po file", that will give you a .pot file that you can put in 10n_cn_small_business/i18n_extra/l10n_cn_small_business.pot  Next to that you could add a file for English as well e.g.  (or another language)  (but I don't know e.g. in which languages you can report)

**Comment by jco-odoo** [comment]
@Colinliz Accounts in Chinese, maybe we can still do it in stable as it is inconsistent with the taxes being in Chinese, but the accounts not.

**Comment by jco-odoo** [comment]
@Colinliz Do you still want us to continue on this?  (at least for master)  
@GoodERPJeff Do you agree on the proposed changes at least?

**Comment by lap-odoo** [comment]
I'm closing this PR, @Colinliz will create a new one that targets master (+ english translations) and we will review it together. 谢谢

**Pull request closed without merging**